src/test/java/lotto/MoneyTest.java

```diff
@@ -0,0 +1,15 @@
+package lotto;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+import lotto.domain.Money;
+import org.junit.jupiter.api.Test;
+
+public class MoneyTest {
+    @Test
+    void validationTest() {
+        assertThatThrownBy(() -> {
+            Money money = new Money(1200);
+        }).isInstanceOf(IllegalArgumentException.class);
```

Comment:
`Money` 객체를 생성할 때 예외가 발생하는걸 확인하고 싶다면 변수 추출도 생략해도 좋을 것 같아요
```suggestion
        assertThatThrownBy(() -> new Money(1200))
                .isInstanceOf(IllegalArgumentException.class);
```